tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -559,6 +557,16 @@ export const getMetricsCardTimeSelection = createSelector(
       return;
     }
 
+    // Handling Linked Time
+    if (linkedTimeEnabled && linkedTimeSelection) {
```

Comment:
So you moved this below in order to do the formatting. Which clips does the clipping? But, we are also clipping in the card container?

If the linked time that is returned is already clipped then would the ```clipped``` boolean returned in ```maybeClipTimeSelectionView``` always be false?